testnets/cosmosicsprovidertestnet/chain.json

```diff
@@ -33,30 +33,30 @@
   },
   "codebase": {
     "git_repo": "https://github.com/cosmos/gaia",
-    "recommended_version": "v21.0.1",
+    "recommended_version": "v22.0.0-rc0",
```

Comment:
Please add this version to the versions.json file

Reply:
Done, thanks!